### PR TITLE
Add geofire lat-lng coords along with new items in Firebase

### PR DESCRIPTION
**Pull request by jschadaway**
For Tony, primarily.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -70,7 +70,7 @@ dependencies {
     compile 'de.hdodenhof:circleimageview:2.2.0'
 
     // Geofire Dependencies
-    compile 'com.firebase:geofire-java:2.1.3'
+    compile 'com.firebase:geofire-android:2.1.3'
```
Comment (tkixi) [inline app/build.gradle]: seems like a git blame change, but i like it

Comment (jschadaway) [reply app/build.gradle]: no u

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ListItemFragment.java [hunk 2]
```diff
@@ -11,7 +13,15 @@
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.EditText;
-
+import android.widget.Toast;
+
+import com.firebase.geofire.GeoFire;
+import com.firebase.geofire.GeoLocation;
+import com.google.android.gms.maps.CameraUpdateFactory;
```
Comment (tkixi) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/ListItemFragment.java]: why u have this

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ListItemFragment.java [hunk 3]
```diff
@@ -11,7 +13,15 @@
...
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
```
Comment (tkixi) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/ListItemFragment.java]: this is monkey imports

Comment (jschadaway) [reply app/src/main/java/com/teamcaffeine/hotswap/navigation/ListItemFragment.java]: commit just for u bb

**Pull request merged**